src/doom/m_crispy.c

```diff
@@ -43,6 +43,14 @@ multiitem_t multiitem_brightmaps[NUM_BRIGHTMAPS] =
     {BRIGHTMAPS_BOTH, "both"},
 };
 
+multiitem_t multiitem_coloredblood[NUM_COLOREDBLOOD] =
+{
+    {COLOREDBLOOD_OFF, "off"},
+    {COLOREDBLOOD_ON, "on"},
+    {COLOREDBLOOD_BOTH, "blood+puffs"},
```

Comment:
Hm, I'm not sure if we really need this third option or just go e.g. "off" / "blood" / "all"?

Reply:
Would you prefer the lost souls bleeding puffs to be turned on when choosing the "all" setting or the "blood" setting in that case? I am not quite sure if it's a good idea to unite the specter blood and lost souls bleeding puffs under one option, because some people got used to your current implementation (blood+puffs), and might prefer specters to bleed normal blood.

Reply:
Actually, the current implementation only applies puffs as blood to Lost Souls *that are already on the map* only *after changing that setting in the menu*. This is an oversight by me, caused by not removing the code in `M_CrispyToggleColoredblood()`. Now I understand where your comment comes from!

I'd prefer if "blood" only changes blood *color*, as in Woof or DSDA-Doom, and "all" changes blood to puffs and spectre blood.